### ISSUE TITLE
Fix sphinx warnings when generating the documentation

### ISSUE THREAD

**Issue by gnecula** [google/jax#1433]
The documentation generation process (including notebooks) generated 2200+ warnings, which makes the Travis log hard to use. 

Example: https://travis-ci.org/google/jax/jobs/592948199

We should look into how to squash these.

**Comment by IgorWilbert** [comment]
@gnecula using  [this](https://www.sphinx-doc.org/en/master/man/sphinx-build.html) link as a reference I was able to suppress the warnings locally by adding `-Q` at the end of the `sphinx-build -M html docs build` command, so only errors would be logged. However, by checking this [old release](https://github.com/google/jax/tree/fbd307edcdaf46fd7c90e2b03d13bdacf1d1cf34) I noticed the `travis.yml` file is gone. In this case, should we close this issue?
